### ISSUE TITLE
Small GUI issues

### ISSUE THREAD

**Issue by civier** [yjmantilla/sovabids#40]
I tested the GUI. It works really well. Congratulations!

Some small things that can be improved:
1) When editing the mapping files, it should say "Modifying individual participant mapping file:" rather than "Modifying rules file:".
2) Downloaded zip file doesn't have a .zip extension (no extension at all)
3) Files within downloaded zip file have the path home/neuro/sovabids/front/app/_convert/* when sovabids is installed in /home/neuro/sovabids, so I guess only the initial slash is removed from the paths (when in reality, home/neuro/sovabids/front/app/_convert/ should have been removed). Maybe because I'm running it on Linux rather than Windows?

**Comment by yjmantilla** [comment]
@civier 
Regarding the second point Im not sure what is happening, I just tested the GUI and at the download it shows:

![image](https://user-images.githubusercontent.com/36543115/130311254-04a18d8e-8342-4ed8-afee-9caf41ec20ab.png)


That is, it has the .zip extension. Is this an linux-only issue?

**Comment by yjmantilla** [comment]
For the download we use:

```python

    response = Response(s.getvalue(), content_type = "application/x-zip-compressed")

```

So is possible that some browsers identify the content type and add the .zip by themselves. The filename is missing too so I guess that my browser just names it as 'downloads'.

I will inquiry in the matter

**Comment by yjmantilla** [comment]
@civier in the last version of the front I think I fixed the filename issue, let me know if it is resolved on your end

**Comment by yjmantilla** [comment]
> Files within downloaded zip file have the path home/neuro/sovabids/front/app/_convert/* when sovabids is installed in /home/neuro/sovabids, so I guess only the initial slash is removed from the paths (when in reality, home/neuro/sovabids/front/app/_convert/ should have been removed). Maybe because I'm running it on Linux rather than Windows?

@civier 
Could you provide a snapshot of where and/or how are you seeing this. Im not sure what is the problem. Is it mainly an aesthetic issue regarding not having the '/'  at the start in linux or does this affects the zip file? Are you able to decompress it ?

**Comment by yjmantilla** [comment]
For the first issue of renaming the mapping annotation, here is what will currently look like:

![image](https://user-images.githubusercontent.com/36543115/130314651-c6e1fc20-24ac-4d4f-aec1-a58f9034bb9b.png)

**Comment by civier** [comment]
No. It is not an aesthetic issue. I CAN extract the files, but they are all
extracted to
CURRENT_DIR/home/neuro/sovabids/front/app/_convert/*
(CURRENT_DIR being the working directory)

Because /home/neuro/sovabids/front/app/_convert/*
is probably where the converted files are kept on the server where the GUI
is running, I assume it's a bug of some sort.

**Comment by yjmantilla** [comment]
Ahh I understand the issue now. In windows this doesn't happen or at least when I extract with 7z (i just press the extract here dialog). I will need to dig in this matter then
